modules/ROOT/pages/dw-core-functions-distinctby.adoc

```diff
@@ -68,69 +73,71 @@ output application/json
 ----
 
 
-
 [[distinctby2]]
+== distinctBy(Null, (item: Nothing, index: Nothing) -> Any): Null
+
+Helper function that allows *distinctBy* to work with null values.
+
+
+[[distinctby3]]
 == distinctBy({ (K)?: V }, (value: V, key: K) -> Any): Object
 
-Returns an object with unlike key value pairs.
+Removes duplicate key-value pairs from an Object.
```

Comment:
Why capitalize Object? The names of specific objects are capitalized, but why "object"?